marconi-chain-index/src/Marconi/ChainIndex/Experimental/Run.hs

```diff
@@ -20,10 +21,20 @@ import Marconi.ChainIndex.Utils qualified as Utils
 import Marconi.Core.Experiment qualified as Core
 import System.Directory (createDirectoryIfMissing)
 import System.Exit (exitFailure)
+import Text.Pretty.Simple (pShowDarkBg)
 
 run :: Text -> IO ()
 run appName = do
+  trace <- defaultStdOutLogger appName
+
+  logInfo trace $ appName <> "-" <> Text.pack Cli.getVersion
+
   o <- Cli.parseOptions
+
+  logInfo trace . Text.toStrict $ pShowDarkBg o
```

Comment:
What is that?

Reply:
This line logs the parsed options at the info level, in a way that is consistent with `Marconi.Sidechain.Run.run`.

I assumed that in `marconi-sidechain` the use of [pShowDarkBg](https://hackage.haskell.org/package/pretty-simple-4.1.2.0/docs/Text-Pretty-Simple.html#v:pShowDarkBg) was intentional for style purposes. `pShowDarkBg` aliases `pShow`, so we could swap `pShow` in without changing the log output.

Does that answer your question, or were you referring to some other part of the line?

Reply:
Why not just `logInfo trace $ Text.pack $ show o`? I think that's what we do in other parts.